docs/.vitepress/config.ts

```diff
@@ -37,20 +31,16 @@ export default defineConfig({
   },
 
   head: [
-    ['meta', { property: 'og:type', content: 'website' }],
-    ['meta', { property: 'og:title', content: ogTitle }],
-    ['meta', { property: 'og:image', content: ogImage }],
-    ['meta', { property: 'og:url', content: ogUrl }],
-    ['meta', { property: 'og:description', content: description }],
-    ['meta', { name: 'twitter:card', content: 'summary_large_image' }],
-    [
-      'script',
-      {
-        async: '',
-        'data-website-id': 'c1840c18-a12c-4a45-a848-55ae85ef7915',
-        src: 'https://umami.aklinker1.io/script.js',
-      },
-    ],
+    meta('og:type', 'website'),
+    meta('og:title', ogTitle),
+    meta('og:image', ogImage),
+    meta('og:url', ogUrl),
+    meta('og:description', description),
+    meta('twitter:card', 'summary_large_image', true),
```

Comment:
Don't really like this third argument here, it's not clear just from looking at it what the third argument is doing. Not a huge problem, since it will probably never be used again.. but I would prefer two functions, `metaProperty` and `metaName` or just have the key be the first argument, like `meta('name', 'twitter:card', 'summary_large_image')`. Or make the third argument an object so the boolean can be labeled: `meta('twitter:card', 'summary_large_image', { useName: true })`